pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -61,9 +61,9 @@ type EventListenerSpec struct {
 // and TriggerTemplate; TriggerBinding provides extracted values for
 // TriggerTemplate to then create resources from.
 type EventListenerTrigger struct {
-	Binding  *EventListenerBinding `json:"binding"`
-	Template EventListenerTemplate `json:"template"`
-	Params   []pipelinev1.Param    `json:"params,omitempty"`
+	Bindings []*EventListenerBinding `json:"bindings"`
+	Template EventListenerTemplate   `json:"template"`
+	Params   []pipelinev1.Param      `json:"params,omitempty"`
```

Comment:
Should we remove `params` from the EventListener? I think that this change to allow multiple TriggerBindings has overlapping functionality with the EventListener params. Now a user can use a TriggerBinding instead of the EventListener params, and I think that it might be confusing to have both options.

The direction that this works seems to be heading towards is embedding TriggerBindings directly in an EventListener yaml (which would completely replace the params).

Reply:
As mentioned in the Triggers WG meeting, I think this is a good idea!

I think that should be done in a separate change though, just to minimize size and the breaking changes of this PR. Happy to file an issue for this once this PR is submitted! ^_^

Reply:
+1 to both  -- let's remove Params in a diff PR!